src/quality_result_gui/quality_errors_tree_model.py

```diff
@@ -303,10 +293,10 @@ def __init__(self, parent: Optional[QObject] = None) -> None:
             QualityErrorTreeItemType.HEADER,
         )
         # Show error priority rows always
-        for priority in list(QualityErrorPriority):
+        for priority in [1, 2, 3]:
             priority_item = QualityErrorTreeItem(
-                [priority, None],
-                str(priority.value),
+                [QualityErrorPriority(priority), None],
+                str(QualityErrorPriority(priority).value),
```

Comment:
This change should be reverted back as QualityErrorPriority enum should work like it did before